Gems/PhysX/PhysX5/Source/Platform/Linux/PhysX_Traits_Platform.h

```diff
@@ -7,5 +7,4 @@
  */
 #pragma once
 
-#define AZ_TRAIT_PHYSX_FORCE_LOAD_MODULES 0
```

Comment:
Since the `AZ_TRAIT_PHYSX_FORCE_LOAD_MODULES` was the only trait in the source code?
Instead of making a bunch of copies of empty the platform specific traits files, all these files could just not be added.

That also removes the need to specify the path to platform specific headers in the `Platform/<platform>/platform_<platform_lower>_files.cmake` files as well.

Reply:
Seems like it, I can remove the trait headers in that case